docs/usage/cli/filter.rst

```diff
@@ -13,40 +13,51 @@ augur filter
     :prog: augur
     :path: filter
 
-How we subsample sequences in the zika-tutoral
-==============================================
+Guides
+======
+
+Below are some examples of using ``augur filter`` to sample data.
+
+Filtering
+---------
 
-As an example, we'll look that the ``filter`` command in greater detail using material from the :doc:`Zika tutorial <docs.nextstrain.org:tutorials/creating-a-workflow>`.
 The filter command allows you to selected various subsets of your input data for different types of analysis.
 A simple example use of this command would be
 
 .. code-block:: bash
 
-	augur filter --sequences data/sequences.fasta --metadata data/metadata.tsv --min-date 2012 --output filtered.fasta
+  augur filter \
+    --sequences data/sequences.fasta \
+    --metadata data/metadata.tsv \
+    --min-date 2012 \
+    --output-sequences filtered_sequences.fasta \
+    --output-metadata filtered_metadata.tsv
 
 This command will select all sequences with collection date in 2012 or later.
 The filter command has a large number of options that allow flexible filtering for many common situations.
 One such use-case is the exclusion of sequences that are known to be outliers (e.g. because of sequencing errors, cell-culture adaptation, ...).
-These can be specified in a separate file:
+These can be specified in a separate text file (e.g. ``exclude.txt``):
 
-.. code-block:: bash
+.. code-block::
 
-    BRA/2016/FC_DQ75D1
-    COL/FLR_00034/2015
-    ...
+  BRA/2016/FC_DQ75D1
+  COL/FLR_00034/2015
+  ...
 
-To drop such strains, you can pass the name of this file to the augur filter command:
+To drop such strains, you can pass the filename to ``--exclude``:
 
 .. code-block:: bash
 
-  augur filter --sequences data/sequences.fasta \
-             --metadata data/metadata.tsv \
-             --min-date 2012 \
-             --exclude config/dropped_strains.txt \
-             --output filtered.fasta
+  augur filter \
+    --sequences data/sequences.fasta \
+    --metadata data/metadata.tsv \
+    --min-date 2012 \
+    --exclude exclude.txt \
+    --output-sequences filtered_sequences.fasta \
+    --output-metadata filtered_metadata.tsv
 
-(To improve legibility, we have wrapped the command across multiple lines.)
-If you run this command (you should be able to copy-paste this into your terminal) on the data provided in the :doc:`Zika tutorial <docs.nextstrain.org:tutorials/creating-a-workflow>`, you should see that one of the sequences in the data set was dropped since its name was in the ``dropped_strains.txt`` file.
+Subsampling
```

Comment:
I just noticed that the sphinx build gives the same URL to this "subsampling" heading nested under "guides" as it does to the CLI's options for subsampling, so clicking the link for the guide in the table of contents take you to the CLI options.

Reply:
Fixed in 3f58fc32.